.github/workflows/nightly-ci-alpha-release.yml

```diff
@@ -0,0 +1,152 @@
+# This is a temporary workflow to release the nightly packages off of the features/conditional-compilation branch.
+# This branch currently has the beta features enabled, so we'd like the alpha packages to be off of that.
+# Note that `main` is currently *not* automaticaly merged into `features/conditional-compilation`, so
+# new alpha releases will only be created when we explicitly merge `main` into the branch (or land an unrelated PR).
+
+name: Nightly CI/CD - feature/conditional-compilation branch
+on:
+  schedule:
+    # minute 0, hour 1 UTC (which is 7pm PST/ 6pm PDT), any day of month, any month, any day of the week
+    # if we want to support only Mon - Fri we need to change the check how we look for new changes. Currently we
+    # check for any new changes in the last 24 hours regardless of day)
+    - cron: '0 1 * * *'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  checkForChanges:
+    name: Check for new changes
+    outputs:
+      hasChanged: ${{ steps.checkChange.outputs.hasChanged }}
+    runs-on: ubuntu-latest
+    steps:
+      # Check-out repo
+      - uses: actions/checkout@v2
+        with:
+          ref: feature/conditional-compilation
```

Comment:
Is there value in this being a new workflow? (and not just put this line `ref: feature/conditional-compilation` in the existing file?) We do CI after every PR to main so I don't think we want a nightly CI as well (really the `nightly-ci` workflow should be renamed `nightly-cd`)

Reply:
Yeah, I realize we don't actually get any extra coverage in the nightly CI (!).